c++/src/StripeStream.hh

```diff
@@ -19,6 +19,7 @@
 #ifndef ORC_STRIPE_STREAM_HH
 #define ORC_STRIPE_STREAM_HH
 
+#include "io/Cache.hh"
```

Comment:
What about using forward declaration of `ReadRangeCache` instead of adding the header file?

Reply:
Good idea